src/controllers/customercontroller.js

```diff
@@ -0,0 +1,70 @@
+class CustomerController extends Controller
+{
+    view()
+    {
+        this._customerHelper();
+    }
+
+    generateOrder()
+    {
+        var products = this.constructor._makeOrder();
```

Comment:
I assume this is from the parent class? You can just call `this._makeOrder()` in that case, no need for `this.constructor`.